### PR TITLE
add new function in report and add jupyter notebook

### PR DESCRIPTION
**Pull request by chengren**
1. function to add and delete one entry in report.py
2. Add one jupyter notebook for city comparison

### CODE REVIEW

File: sss/report.py [hunk 1]
```diff
@@ -89,10 +89,75 @@ def report_to_db(path, db_url= default_db_url):
...
+    session.commit()
+    session.close()
 
 
+def delete_one_entry_reportdb(year,
+                              state,
+                              analysis_type,
+                              db_url=default_db_url):
+    """
+    This function delete one record. In case some records were insert accidently.
```
Comment (bhazelton) [inline sss/report.py:144]: "insert accidently" -> "inserted accidentally"

**Review by bhazelton — COMMENTED** [review]
Let's move the puma equivalency pdf file to the "docs" folder instead of a new folder inside "sss".

Similarly, let's move the poverty_th.csv to the sss/data folder

I think the city_comparison notebook looks great!

In the "data_preprocess_formap" notebook:
- First: restart your kernel and run all the cells in order. I think you'll find some problems. You'll need to change the location of the poverty_th.csv file for one.
- Remove any unnecessary cells, like where you change directories
- After any changes, restart the kernel and run all cells again to make sure things are consistent.

After Hector's changes to pass files rather than urls is merged, we will want to update these and other notebooks to pass files rather than urls, but if this PR is ready first let's get it in and worry about that consistency issue later.

**Comment by chengren** [comment]
> Let's move the puma equivalency pdf file to the "docs" folder instead of a new folder inside "sss".
> 
> Similarly, let's move the poverty_th.csv to the sss/data folder
> 
> I think the city_comparison notebook looks great!
> 
> In the "data_preprocess_formap" notebook:
> 
> * First: restart your kernel and run all the cells in order. I think you'll find some problems. You'll need to change the location of the poverty_th.csv file for one.
> * Remove any unnecessary cells, like where you change directories
> * After any changes, restart the kernel and run all cells again to make sure things are consistent.
> 
> After Hector's changes to pass files rather than urls is merged, we will want to update these and other notebooks to pass files rather than urls, but if this PR is ready first let's get it in and worry about that consistency issue later.

Fixed the issues mentioned above

**Review by bhazelton — APPROVED** [review]
Looks good!

**Pull request merged**